rollup.config.js

```diff
@@ -32,7 +32,7 @@ if (process.env.NODE_ENV === 'production') {
 export default [
     // Rollup lib
```

Comment:
For consistency this should be updated to `// Rollup src`

```suggestion
    // Rollup src
```